src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/BaseProjectConfigurationDimensionProvider.cs

```diff
@@ -75,6 +76,30 @@ private async Task<ImmutableArray<string>> GetOrderedPropertyValuesAsync(Unconfi
             }
         }
 
+        /// <summary>
+        /// Gets the property values for the dimension.
+        /// </summary>
+        /// <param name="project"><see cref="Project"/>.</param>
+        /// <returns>Collection of values for the dimension.</returns>
+        /// <remarks>
+        /// From <see cref="IProjectConfigurationDimensionsProvider"/>.
+        /// </remarks>
+        private ImmutableArray<string> GetOrderedPropertyValues(Project project)
```

Comment:
Can `GetOrderedPropertyValuesAsync` be rewritten to use this method, to avoid the code duplication this introduces?